src/currencyRatesFileGen.js

```diff
@@ -0,0 +1,59 @@
+// prebidCurrencyRatesFileGenerator
+// loads currency rates from api.fixer.io and creates a JSON representation in S3
+// revision:
+var http = require('http');
+var AWS = require('aws-sdk');
+
+var fromCurrencies = ['USD','GBP'];
+var supportedCurrencies = 'AUD,BRL,CAD,CHF,CNY,CZK,DKK,EUR,GBP,HKD,HUF,IDR,ILS,INR,JPY,KRW,MXN,MYR,NOK,NZD,PHP,PLN,RUB,SEK,SGD,THB,TRY,USD,ZAR';
+
+var bucket = 'currency.prebid.org';
+var filename = 'latest.json';
+var debug = true;
+var expires = 24 * 3600 + 5; // when to expire for HTTP "Expires:" header (seconds)
+
+exports.handler = function(event, context) {
```

Comment:
I wasn't sure what this was - probably a good idea to mention this is part of the AWS/Lambda framework / JSDOC it